src/lgdo/lh5/tools.py

```diff
@@ -171,6 +174,26 @@ def show(
 
         print(f"{indent}{char} \033[1m{key}\033[0m · {dtype} {_attrs}")  # noqa: T201
 
+        if detail and isinstance(val, h5py.Dataset):
+            char = "|       "
+            if killme:
+                char = "        "
+            toprint = f"{indent}{char}"
+            try:
+                toprint += f"\x1b[3mdtype\x1b[0m {val.dtype}"
```

Comment:
Uhm I don't see the colors in my terminal, but I do see the bold text. I'm wondering how much this ANSI sequence is supported. I would suggest to just print unformatted text (and maybe with a separator like `numchunks=246`).

Reply:
It shows in italics on my terminal. I changed the escape character to the other one present in the file (IDK if that matters) and added a separator. I would leave it otherwise so the italics can work for whoever it works for and it shows unformatted for others.

Reply:
Ok! Yeah italics are not working on my terminal for some reason.